tests/setup/enzymeadapter.js

```diff
@@ -110,7 +110,16 @@ export default class AnswersAdapter extends EnzymeAdapter {
           composed: args.composed,
           cancelable: args.cancelable
         });
-        Object.assign(event, args);
+
+        if (Array.isArray(...args)) {
```

Comment:
what vulnerability is this change related to?

Reply:
This never worked and there were no usage of this case so it didn't surface. But the test I added need it.